near-bindgen-core/src/initializer_attribute.rs

```diff
@@ -54,6 +54,13 @@ pub fn process_init_method(
     impl_type: &Type,
     is_trait_impl: bool,
 ) -> syn::Result<TokenStream2> {
+    let attrs = method.attrs.iter().fold(TokenStream2::new(), |mut acc, attr| {
+        let attr_str = attr.path.to_token_stream().to_string();
+        if attr_str != "callback_args_vec".to_string() && attr_str != "callback_args".to_string() {
```

Comment:
You don't need `&str.to_string()` to compare. `if attr_str != "callback_args_vec" && attr_str != "callback_args" {` should be just fine.

Reply:
Fixed.